src/index.scss

```diff
@@ -26,3 +22,8 @@ main {
 .bx--content {
   padding: 0;
 }
+
+$feature-flags: (
```

Comment:
The feature-flags mapping should come before the styles import.
See https://www.carbondesignsystem.com/tutorial/react/step-2/#install-grid